backend/prompt_studio/prompt_studio/models.py

```diff
@@ -21,6 +21,7 @@ class EnforceType(models.TextChoices):
         BOOLEAN = "boolean", "Response sent as boolean"
         JSON = "json", "Response sent as json"
         TABLE = "table", "Response sent as table"
+        RECORD = "record", "Response sent for records"
```

Comment:
@harini-venkataraman what's the difference between table and record? Can you document that as well?

Reply:
@chandrasekharan-zipstack Yes, will add it to the description. 
Records are enforce type added to extract data where the entries are present sequentially for a single record, unlike tables.